integration/test/ecs/ecs_metadata/ecs_metadata_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build integration
+// +build integration
+
+package ecs_metadata
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"testing"
+	"time"
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+)
+
+// Purpose: Detect the changes in metadata endpoint for ECS Container Agent https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/util/ecsutil/ecsutil.go#L67-L75
```

Comment:
To be more specifically a bit, during the [scrapping  metadata endpoint](https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/util/ecsutil/ecsutil.go#L72-L79),  it would extract the Region, ClusterName, TaskARN. After extracting those variables, it would create a [log group based on that format that includes the ClusterName](https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/translate/logs/metrics_collected/prometheus/ruleLogGroupName.go#L33). So to speak, if the `clustername` which scrapped from `metadataendpointv4` does not exist, then it would fail the test. So to your point, yes. Also if the ECS cluster does have the v5 endpoint but we have not updated that, then it would fail the test based on the aforementioned points.